src/internal/m365/collection/drive/limiter_test.go

```diff
@@ -586,7 +591,7 @@ func (suite *LimiterUnitSuite) TestGet_PreviewLimits_tree() {
 	for _, test := range backupLimitTable(drive1, drive2) {
 		suite.Run(test.name, func() {
 			runGetPreviewLimits(
-				suite.T(),
+				t,
```

Comment:
same here as one of the other comments, could you please change this back to `suite.T()` so test failures are properly reported?